### PR TITLE
Contribution details

### LINKED ISSUE #121: Use contribution date/time from IPFS doc in UI
Contributions should be ordered by their date/time instead of ID. And the date/time should be visible somehow (perhaps day markers between items or something).

### PR DESCRIPTION
**Pull request by skddc**
Adds a details pane for contribution records, similar to contributor profiles.

Just the basic details for now. Can be extended with all kinds of data, links, icons, etc. from here on.

Screenshot from 2019-07-17 12-58-06

closes #121

### CODE REVIEW

**Comment by skddc** [comment]
Just opened https://github.com/67P/kredits-contracts/issues/153 about this.

**Comment by galfert** [comment]
Code looks alright and everything works.

One thing I noticed though: when scrolling down and selecting a contribution further down, you can't see the contribution details, because those are still at the top of the page.

Wouldn't it be better to show the contribution details right below the selected contribution?

**Comment by skddc** [comment]
> Wouldn't it be better to show the contribution details right below the selected contribution?

No. The solution is to have the details pane be fixed, but that's a major layout problem, which I wasn't able to solve quickly. This is just the basics for contributions details, as described above. All improvements from here on are separate PRs (and btw, anyone can help with them).

File: app/styles/components/_contribution-details.scss [hunk 1]
```diff
@@ -0,0 +1,60 @@
+section#contribution-details {
+  header {
+    nav {
+      .amount {
+        font-size: 1.5rem;
+      }
+
+      .symbol {
+        // font-size: 0.8rem;
```
Comment (skddc) [inline app/styles/components/_contribution-details.scss]:
```suggestion
```

**Comment by galfert** [comment]
👍

**Pull request merged**